projects/js-packages/components/components/boost-score-graph/tooltip.tsx

```diff
@@ -0,0 +1,87 @@
+import { getScoreLetter } from '@automattic/jetpack-boost-score-api';
+import { dateI18n } from '@wordpress/date';
+import { __, sprintf } from '@wordpress/i18n';
+import { type FunctionComponent } from 'react';
+import './style-tooltip.scss';
+import { Period } from '.';
+
+export const Tooltip: FunctionComponent = ( { period }: { period: Period } ) => {
```

Comment:
I'm probably not used to the syntax, but - why not just expect for `period: Period` here?

```js
{ period }: { period: Period }
```

Reply:
If we had done `{ period: Period }`, it would not be considered as the type. It would rename the `period` property to `Period` for the context of this function.